### PR TITLE
RTB House: update parameters

### PR DESCRIPTION
**Pull request by piwanczak**
PR addresses stale / non up to date params requirements by RTB House adapter

### CODE REVIEW

File: static/bidder-params/rtbhouse.json [hunk 1]
```diff
@@ -1,8 +1,13 @@
...
-  "required": []
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "RTB House Adapter Params",
+    "description": "A schema which validates params accepted by the RTB House adapter",
+    "type": "object",
+    "properties": {
+		"publisherId": {
+			"type": "string",
+            "description": "The publisher’s ID provided by RTB House"
+        }
```
Comment (SyntaxNode) [inline static/bidder-params/rtbhouse.json:10]: The formatting of this document is a bit misaligned. Could you please run it through a "beautifier"? ... or align the whitespace of the `properties` object.

Comment (piwanczak) [reply static/bidder-params/rtbhouse.json:10]: converted tabs to spaces. Please have a look if it's fine? If you prefer any other linters - happy to adjust

Comment (SyntaxNode) [reply static/bidder-params/rtbhouse.json:10]: That works. Thank you.

**Review by SyntaxNode — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**